javalib/src/main/scala/java/util/AbstractQueue.scala

```diff
@@ -0,0 +1,29 @@
+package java.util
+
+abstract class AbstractQueue[E] protected ()
+    extends AbstractCollection[E]
+    with Queue[E] {
+
+  override def add(e: E): Boolean =
+    if (offer(e)) true
+    else throw new IllegalStateException()
+
+  def remove(): E =
+    if (!isEmpty()) poll()
+    else throw new NoSuchElementException()
+
+  def element(): E =
+    if (!isEmpty()) peek()
+    else throw new NoSuchElementException()
+
+  override def clear(): Unit =
+    while (poll()!=null) {}
```

Comment:
Spaces around `!=` (I guess this could be taught to Scalastyle).
Also, the `while` kind of makes me want `{}` around the method body, although it's a one-line.